tests/generation/test_generation_tf_logits_process.py

```diff
@@ -127,29 +136,21 @@ def _check_repetition_penalty_outputs(self, scores, vocab_size):
         self.assertAlmostEqual(scores[1, 5].numpy(), (4 / vocab_size) / 2)
         self.assertAlmostEqual(scores[0, 2].numpy(), (1 / vocab_size))  # unused tokens should see no change
 
-    def test_repetition_penalty_dist_process(self):
-        vocab_size, cur_len, input_ids, scores = self._get_repetition_penalty_inputs()
-        rep_penalty_proc = TFRepetitionPenaltyLogitsProcessor(penalty=2.0)
-        scores = rep_penalty_proc(input_ids, tf.identity(scores), cur_len)
-        self._check_repetition_penalty_outputs(scores, vocab_size)
-
-    def test_repetition_penalty_dist_process_xla(self):
-        vocab_size, cur_len, input_ids, scores = self._get_repetition_penalty_inputs()
-        rep_penalty_proc = TFRepetitionPenaltyLogitsProcessor(penalty=2.0)
-        rep_penalty_proc = tf.function(rep_penalty_proc, jit_compile=True)  # added line wrt non-XLA test
-        scores = rep_penalty_proc(input_ids, tf.identity(scores), cur_len)
-        self._check_repetition_penalty_outputs(scores, vocab_size)
-
-    def test_top_k_dist_warper(self):
+    @parameterized.expand([(False,), (True,)])
+    def test_top_k_dist_warper(self, use_xla):
         input_ids = None
         vocab_size = 10
         batch_size = 2
 
         # create ramp distribution
-        ramp_logits = np.broadcast_to(np.arange(vocab_size)[None, :], (batch_size, vocab_size)).copy()
+        ramp_logits = np.broadcast_to(
+            np.arange(vocab_size, dtype=np.float32)[None, :], (batch_size, vocab_size)
```

Comment:
```suggestion
            np.arange(vocab_size, dtype=np.float32), (batch_size, vocab_size)
```
You'll never win at [Sasha's Torch golf](https://twitter.com/srush_nlp/status/1516781757596680194) if you manually add dimensions that broadcasting would handle for you!

(Yes, this is the most pointless nit I've ever suggested in a code review)

Reply:
😂 Masters of tensor manipulation